query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -203,6 +203,12 @@ private String convertFilterToString(
           break;
         default:
           rhs = handleValueConversionForLiteralExpression(filter.getLhs(), filter.getRhs());
+
+          Optional<String> valueFilter =
+              getValuesFilterForMapColumn(
+                  filter.getLhs(), rhs, filter.getOperator(), paramsBuilder, executionContext);
+          valueFilter.stream().forEach(str -> builder.append(str));
```

Comment:
```suggestion
          valueFilter.ifPresent(builder::append);
```
since there is no collection within Optional

Reply:
Done.